tools/ci/ubuntu-mirror-list.txt

```diff
@@ -0,0 +1,40 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+################################################################################
+
+# This mirror list can be used in CI to set up the Docker containers without only
+# relying on the default Ubuntu mirror provided by Azure.
+# FYI: apt ignores lines that are empty or start with '#'
```

Comment:
I did a brief check of the apt code on that one: https://github.com/Debian/apt/blob/b3a5bbe234d4206b7736bca2ca7f56feaa92f5ff/methods/mirror.cc#L242